app/views/settings/from_address/index.html.erb

```diff
@@ -9,19 +9,28 @@
 
     <% if @from_address.pending? && @from_address.errors.blank? %>
       <%= govuk_notification_banner(title_text: t('notification_banners.important'), classes: "govuk-!-margin-top-6") do %>
-        <%= @presenter.message.html_safe %>
-
-        <p>
-          <span role="alert"></span>
-          <%= tag.button t('settings.from_address.resend_link_text'),
-            class: 'fb-link-button',
-            data: {
-              remote: true,
-              method: 'post',
-              url: api_service_settings_from_address_resend_path(service.service_id),
-              success: t('settings.from_address.link_resent_success')
-            } %>
-        </p>
+        <% if @from_address.allowed_domain?(@from_address.decrypt_email) %>
+          <%= @presenter.message.html_safe %>
+            <p>
+              <span role="alert"></span>
+              <%= tag.button t('settings.from_address.resend_link_text'),
+                class: 'fb-link-button',
+                data: {
+                  remote: true,
+                  method: 'post',
+                  url: api_service_settings_from_address_resend_path(service.service_id),
+                  success: t('settings.from_address.link_resent_success')
+                } %>
+            </p>
+        <% else %>
+          <%= t('warnings.from_address.settings.contact_us').html_safe %>
+            <p>
+              <span role="alert"></span>
```

Comment:
I need to check why this empty `<span>` is here. It has an alert role, which indicates we're using JS to inject content we want announced by screenreaders... but I'm not sure that's true anymore.

Reply:
Just checked further.  This empty span is defintily redundant in the 'contact us' message.

In the block above, the 'resend email' link is done via ajax, and on success we udate the hidden span with the success message the `role=alert` means it gets read out by screenreaders.

Reply:
Ahh I see, have removed the redundant `span` in the contact us message.